pywr/nodes.py

```diff
@@ -861,4 +861,61 @@ def after(self, timestep):
         # update flow on transfer node to flow via link node
         self.commit_all(self.link.flow)
 
+
+class DelayNode(Node):
+    """ A node that delays flow for a given number of timesteps or days.
+
+    This is a composite node consisting internally of an Input and an Output node. A
+    `FlowDelayParameter` is used to delay the flow of the output node for a given period prior
+    to this delayed flow being set as the flow of the input node. Connections to the node are connected
+    to the internal output node and connection from the node are connected to the internal input node
+    node.
+
+    Parameters
+    ----------
+    model : `pywr.model.Model`
+    name : string
+        Name of the node.
+    timesteps: int
+        Number of timesteps to delay the flow.
+    days: int
+        Number of days to delay the flow. Specifying a number of days (instead of a number
+        of timesteps) is only valid if the number of days is exactly divisible by the model
+        timestep delta.
+    initial_flow: float
+        Flow value from node when model timestep is less than or equal to the number of delay
```

Comment:
I think this description could be clearer.

Reply:
` Flow provided by node for initial timesteps prior to any delayed flow being available.
  This is constant across all delayed timesteps and any model scenarios. Default is 0.0.`

this any clearer?

Reply:
Yup